src/ray/rpc/worker/direct_actor_client.h

```diff
@@ -115,9 +118,6 @@ class DirectActorClient : public std::enable_shared_from_this<DirectActorClient>
   std::deque<std::pair<std::unique_ptr<PushTaskRequest>, ClientCallback<PushTaskReply>>>
       send_queue_;
 
-  /// The next sequence number to assign to a task for this server.
-  int64_t next_seq_no_ = 0;
-
   /// The number of bytes currently in flight.
```

Comment:
Shall we add GUARDED_BY(mutex_) annotations as appropriate here? Just need to '#include <absl/base/thread_annotations.h>'

Reply:
Do we have any absl dependencies right now? I think in the past we've tried to only have dependencies on stdlib, but I'm okay with adding it if we don't care anymore.

Reply:
IIRC we already include the absl::base as a dependency, so you should be able to add the annotation without needing to change the BUILD file.

Reply:
Yes we do use absl and I am pulling in absl/random for a new PR so wouldn't worry about adding it.